File: backend/App/Core/HttpHeader.php
<?php

declare(strict_types=1);

namespace App\Core;

enum HttpHeader: string
{
    case Json = 'application/json';
    case PlainText = 'text/plain';

    public function header(): void
    {
        header("Content-Type: {$this->value}; charset=UTF-8");
    }
}

File: tests/http_header.py
from enum import StrEnum


class HttpHeader(StrEnum):
    """
    Enum for common content types used in http responses.
    """

    HTML_TEXT = "text/html"
    PLAIN_TEXT = "text/plain"

    def __init__(self, content_type: str) -> None:
        self.content_type = f"{content_type}; charset=UTF-8"
